.github/actions/run-local-api/action.yml

```diff
@@ -1,7 +1,10 @@
-name: Run E2E tests
-description: Run the E2E tests against an environment
+name: Run Core API image in Docker
+description: Runs the Core API image in Docker, making it available for E2E testing
 
 inputs:
+  api-image:
+    description: API image to use
```

Comment:
I think it would be good to provide more information here. What format is this expected to be in?

Reply:
Example value added.